src/gurobi_optimods/min_cut.py

```diff
@@ -0,0 +1,289 @@
+"""
+Minimum Cut
+-----------
+"""
+
+import logging
+from typing import Optional, overload
+
+import gurobipy as gp
+import gurobipy_pandas as gppd
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+from gurobipy import GRB
+
+try:
+    import networkx as nx
+except ImportError:
+    nx = None
+
+from gurobi_optimods.utils import optimod
+
+logger = logging.getLogger(__name__)
+
+
+@overload
```

Comment:
The overloads can be removed for both mods. I thought they would be useful for documentation, but they just clutter things up at the moment.